test/integration/08-specter.spec.ts

```diff
@@ -3,17 +3,27 @@
  */
 import { getChainBalance } from "lightning"
 import mongoose from "mongoose"
+import { yamlConfig } from "src/config"
 import { bitcoindAccountingPath } from "src/ledger/ledger"
 import { getActiveOnchainLnd } from "src/lndUtils"
 import { baseLogger } from "src/logger"
 import { MainBook, setupMongoConnection } from "src/mongodb"
+import { SpecterWalletConfig } from "src/types"
 import { SpecterWallet } from "src/SpecterWallet"
 import { UserWallet } from "src/userWallet"
 import { bitcoindDefaultClient, sleep } from "src/utils"
 import { checkIsBalanced, mockGetExchangeBalance, RANDOM_ADDRESS } from "./helper"
 
 let specterWallet
 
+const specterWalletConfig: SpecterWalletConfig = {
```

Comment:
I explicitly made a new config here instead of importing the one made in the `src/entrypoint/cron.ts` file